stock_landed_costs_invoice/__openerp__.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "WMS Landed Costs - Invoice",
+
+    'summary': """
+        allow to creating & grouping invoices per supplier of all
+        selected landed costs.
+""",
+
+    'description': """
+        TODO
+    """,
+
+    'author': "Elico Corp / Camptocamp",
+    'website': "",
+
+    'category': 'Warehouse',
+    'version': '0.1',
```

Comment:
Please check the guidelines on how to define the module version number.
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers